x-pack/solutions/security/plugins/security_solution/public/detections/containers/detection_engine/rules/use_rule_from_timeline.test.ts

```diff
@@ -96,7 +101,10 @@ const selectedTimeline: ResolveTimelineResponse = {
   },
 };
 
-describe('useRuleFromTimeline', () => {
+// TODO: come back to this test later, no idea why it fails with the new picker, even when I mock all the hooks
+// probably depends on some weird timing or other non-deterministic behavior.
+// https://github.com/elastic/security-team/issues/11959
+describe.skip('useRuleFromTimeline', () => {
```

Comment:
Do we still need to skip this one if we don't have the feature flag enabled?

Reply:
I just tested locally and it seems to be fine

<img width="1031" height="161" alt="image" src="https://github.com/user-attachments/assets/671793e8-91f8-4c60-8985-87be728c0351" />

Reply:
Unskipped [here](https://github.com/elastic/kibana/pull/227422/commits/e6ca1dca90308a190bd914fbfb91795cf02c73e2)